### PR TITLE
[All hosts] (web viewer) article about the web viewers used by add-ins

### PR DESCRIPTION
**Pull request by Rick-Kirkham**

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[5947414](https://github.com/OfficeDev/office-js-docs-pr/pull/960/commits/59474148f02ef1e491a856c90834b4326bb7ed85)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/web-viewers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-viewer/docs/concepts/web-viewers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-viewer/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905030006508901-960%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=U5FQ60VZmDIqyCxn%2FqGZc%2F4LGl0CWY8%2FmLtPpJoWNDo%3D&st=2019-05-03T00%3A03%3A21Z&se=2019-06-03T00%3A08%3A21Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/concepts/web-viewers-used-by-office-web-add-ins.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: Web viewers used by Office Add-ins
+description: ''
+ms.date: 05/06/2019
```
Comment (AlexJerabek) [inline docs/concepts/web-viewers-used-by-office-web-add-ins.md]: Is this article launching Monday?

File: docs/concepts/web-viewers-used-by-office-web-add-ins.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+|Android|Chrome|
+|Windows / non-subscription Office 2013 or later|Internet Explorer 11\*|
+|Windows 10 ver. < 19*mm* / Office 365|Internet Explorer 11\*|
+|Windows 10 ver. >= 19*mm* / Office 365 ver < 16.0.11629|Internet Explorer 11\*|
+|Windows 10 ver. >= 19*mm* / Office 365 ver >= 16.0.11629|Edge|
+
+\* Internet Explorer 11 does not support JavaScript versions later than ES5. If any of your add-in's users have platforms that use Internet Explorer 11, then to use the syntax and features of ECMAScript 2015 or later, you will need to either transpile your JavaScript to ES5 or use a polyfill. Also, Internet Explorer 11 does not support some HTML 5 features such as media, recording, and location.
+
+> [!NOTE]
+> Until they are generally available, you may need to be a Windows Insider to get a Windows version 19*mm* or greater, and you may need to be an Office Insider to get Office version 16.0.11629 or greater.
```
Comment (AlexJerabek) [inline docs/concepts/web-viewers-used-by-office-web-add-ins.md]: I think removing "may" from this sentence makes more sense. If the versions aren't generally available, they won't have them without Insiders.

File: docs/concepts/web-viewers-used-by-office-web-add-ins.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+>
+> To join Windows Insiders:
+> 
+> 1. Navigate to [Windows Insider](https://insider.windows.com) and click the link to join Windows Insiders.
```
Comment (AlexJerabek) [inline docs/concepts/web-viewers-used-by-office-web-add-ins.md]: Consider changing "Navigate" to "Go" to sound less formal.

File: docs/concepts/web-viewers-used-by-office-web-add-ins.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+> 2. You will be taken to a page with instructions about how to use Windows Settings to enable preview builds of Windows. Follow the instructions. When you select the pace of updates, choose the fastest option.
+>
+> To join Office Insiders
+> 
+> 1. Navigate to [Get started as an Office Insider](https://insider.office.com/en-us/join).
```
Comment (AlexJerabek) [inline docs/concepts/web-viewers-used-by-office-web-add-ins.md]: Remove the localization from the URL.

File: docs/concepts/web-viewers-used-by-office-web-add-ins.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+> [!NOTE]
+> Until they are generally available, you may need to be a Windows Insider to get a Windows version 19*mm* or greater, and you may need to be an Office Insider to get Office version 16.0.11629 or greater.
+>
+> To join Windows Insiders:
+> 
+> 1. Navigate to [Windows Insider](https://insider.windows.com) and click the link to join Windows Insiders.
+> 2. You will be taken to a page with instructions about how to use Windows Settings to enable preview builds of Windows. Follow the instructions. When you select the pace of updates, choose the fastest option.
+>
+> To join Office Insiders
```
Comment (AlexJerabek) [inline docs/concepts/web-viewers-used-by-office-web-add-ins.md]: Missing ':'

File: docs/concepts/web-viewers-used-by-office-web-add-ins.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+\* Internet Explorer 11 does not support JavaScript versions later than ES5. If any of your add-in's users have platforms that use Internet Explorer 11, then to use the syntax and features of ECMAScript 2015 or later, you will need to either transpile your JavaScript to ES5 or use a polyfill. Also, Internet Explorer 11 does not support some HTML 5 features such as media, recording, and location.
+
+> [!NOTE]
+> Until they are generally available, you may need to be a Windows Insider to get a Windows version 19*mm* or greater, and you may need to be an Office Insider to get Office version 16.0.11629 or greater.
+>
+> To join Windows Insiders:
+> 
+> 1. Navigate to [Windows Insider](https://insider.windows.com) and click the link to join Windows Insiders.
+> 2. You will be taken to a page with instructions about how to use Windows Settings to enable preview builds of Windows. Follow the instructions. When you select the pace of updates, choose the fastest option.
```
Comment (AlexJerabek) [inline docs/concepts/web-viewers-used-by-office-web-add-ins.md]: Consider "the pace of updates" -> "the update pace"

File: docs/concepts/web-viewers-used-by-office-web-add-ins.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+|Office Online|The browser in which Office Online is opened.|
+|Mac|Safari|
+|iOS|Safari|
+|Android|Chrome|
+|Windows / non-subscription Office 2013 or later|Internet Explorer 11\*|
+|Windows 10 ver. < 19*mm* / Office 365|Internet Explorer 11\*|
+|Windows 10 ver. >= 19*mm* / Office 365 ver < 16.0.11629|Internet Explorer 11\*|
+|Windows 10 ver. >= 19*mm* / Office 365 ver >= 16.0.11629|Edge|
+
+\* Internet Explorer 11 does not support JavaScript versions later than ES5. If any of your add-in's users have platforms that use Internet Explorer 11, then to use the syntax and features of ECMAScript 2015 or later, you will need to either transpile your JavaScript to ES5 or use a polyfill. Also, Internet Explorer 11 does not support some HTML 5 features such as media, recording, and location.
```
Comment (AlexJerabek) [inline docs/concepts/web-viewers-used-by-office-web-add-ins.md]: Will this be prominent enough without being an !IMPORTANT or something?

File: docs/concepts/web-viewers-used-by-office-web-add-ins.md (hunk 6 above)
Comment (Rick-Kirkham) [reply docs/concepts/web-viewers-used-by-office-web-add-ins.md]: I'm an old fogey about noun-noun constructions, so I'd like to keep it as a prepositional phrase.

**Comment by Rick-Kirkham** [comment]
> @Rick-Kirkham Can we include screenshots of the IE11 vs Edge viewers to quickly determine which viewer is available on someone's computer?

@Reezaali Not sure what you mean. _Both_ browsers are available on Win 10 machines. What determines which one is used by Office add-ins is the Office version and the Windows version. This is explained in the article. I don't know what kind of screenshots would be relevant.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[604a010](https://github.com/OfficeDev/office-js-docs-pr/pull/960/commits/604a01098f1e1c4e7ce11c9cae698f6ea3f4d1ae)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/web-viewers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-viewer/docs/concepts/web-viewers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-viewer/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905031922572045-960%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=FIuSIedasgikfGqMxpwUy5YR7Cc%2BauZNlGoQ5I7XJaY%3D&st=2019-05-03T19%3A19%3A29Z&se=2019-06-03T19%3A24%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/concepts/web-viewers-used-by-office-web-add-ins.md [hunk 8]
```diff
@@ -22,24 +22,25 @@ The following table shows which browser is used for the various platforms and op
...
+> [!IMPORTANT]
+> Internet Explorer 11 does not support JavaScript versions later than ES5. If any of your add-in's users have platforms that use Internet Explorer 11, then to use the syntax and features of ECMAScript 2015 or later, you will need to either transpile your JavaScript to ES5 or use a polyfill. Also, Internet Explorer 11 does not support some HTML 5 features such as media, recording, and location.
 
 > [!NOTE]
-> Until they are generally available, you may need to be a Windows Insider to get a Windows version 19*mm* or greater, and you may need to be an Office Insider to get Office version 16.0.11629 or greater.
+> Until they are generally available, you need to be a Windows Insider to get a Windows version 1903 or greater, and you may need to be an Office Insider to get Office version 16.0.11629 or greater.
```
Comment (AlexJerabek) [inline docs/concepts/web-viewers-used-by-office-web-add-ins.md]: There's one more "may" for the Office Insider section.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3e64fca](https://github.com/OfficeDev/office-js-docs-pr/pull/960/commits/3e64fca5e7a58dcc4716206e750438c1b7b85be6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/web-viewers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-viewer/docs/concepts/web-viewers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-viewer/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905031927253166-960%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=r1HJB3UBf8JJGAk5QedPa6NcEyHvZ6qv5isDdW6FVYA%3D&st=2019-05-03T19%3A23%3A55Z&se=2019-06-03T19%3A28%3A55Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[63d8634](https://github.com/OfficeDev/office-js-docs-pr/pull/960/commits/63d86345c381a7782d9fba93595ab86678e5167b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/web-viewers-used-by-office-web-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-viewer/docs/concepts/web-viewers-used-by-office-web-add-ins.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/edge-viewer/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C3%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201905031958161679-960%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=2D6c4C57dxwBY98NiifVggGHnavjoDzYQCA8lixK%2BKU%3D&st=2019-05-03T19%3A54%3A54Z&se=2019-06-03T19%3A59%3A54Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**